code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -392,8 +401,53 @@
 	if(prob(50))
 		new /obj/item/seeds/random(src) //oops, an additional packet might have slipped its way into the box
 
+/obj/item/storage/box/syndie_kit/pistol
+
+/obj/item/storage/box/syndie_kit/pistol/PopulateContents()
+	new /obj/item/gun/ballistic/automatic/pistol(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+
+/obj/item/storage/box/syndie_kit/pistolammo
+
+/obj/item/storage/box/syndie_kit/pistolammo/PopulateContents()
+	new /obj/item/ammo_box/magazine/m10mm(src)
+	new /obj/item/ammo_box/magazine/m10mm(src)
+
 /obj/item/storage/box/syndie_kit/revolver
 
 /obj/item/storage/box/syndie_kit/revolver/PopulateContents()
 	new /obj/item/gun/ballistic/revolver(src)
 	new /obj/item/ammo_box/a357(src)
+	new /obj/item/ammo_box/a357(src)
+	new /obj/item/ammo_box/a357/ap(src)
+	new /obj/item/ammo_box/a357/ap(src)
+	new /obj/item/ammo_box/a357/rubber(src)
+	new /obj/item/ammo_box/a357/rubber(src)
```

Comment:
The revolver is already stupid powerful. Why do you need to give it a shitload more ammo for free?

Reply:
I'll be really honest here, I think the revolver is a fucking awful purchase at 13tc and primarily it is due to how expensive the ammo is and what you lose out on when you buy it. While the revolver definitely makes a single spaceman horizontal as hell, you run dry of ammo very quickly and you are unable to afford paying for the kinds of utility that actually contribute to success if you start buying speed loaders. Even then, you have sunk 13 tc into this gun so no adrenals. A taser (presently) will get you in one. Don't even pretend sec don't themselves have cheap equipment.

The problems with the revolver being a cheap weapon has nothing on the absolute dumpster fire that is security equipment. Some speed loaders lets the traitor operate in the strengths they were meant to have. Good in prolonged skirmishes. Good in a 1v1. But all this ammo cannot still match against the combined efforts of a fully geared security force. Operating correctly, you will win.

For reference, you have ballistic vests in the armory and those cut revolver damage by more than half, and shields will reliably protect you against most of the bullets, even AP. Once you run out of AP, you start having to print standard ammo for your speed loaders. Which means bodied will always overwhelm a revolver tot not working their arse off.